### DISCUSSION TITLE
Is the rate limit for V1 per-account or per-IP or the rate limit for the entire server?

### DISCUSSION THREAD

**Discussion by renewboy** [acheong08/ChatGPT#1133] [General]

**Comment by singlag** [comment]
should set to limit at 1 requests / 10 seconds for both account and IP
It is enough for personal non business use due to it is a free api

**Comment by acheong08** [comment]
My current solution is too keep adding servers until I go bankrupt. Will lower rate limit once I can get a commercial tier out.

**Comment by acheong08** [comment]
Currently per IP

**Comment by youmebangbang** [comment]
@acheong08 You have a crypto address for donation towards this project?

**Comment by acheong08** [comment]
I have a ton of money in crypto but I don't know how to withdraw. In USDT

**Comment by acheong08** [comment]
Kraken suspended my account without reason

**Comment by acheong08** [comment]
Everything stuck in cold wallet

**Comment by youmebangbang** [comment]
That sucks, hopefully you can get things worked out with kraken.

**Comment by acheong08** [comment]
@youmebangbang You seem well versed in crypto. What are some exchanges I could use to get my USDT into bank or something?

Has to work in Malaysia

**Comment by licett** [comment]
You  may try depay (www.depay.one) to change your USDT into USD with a 2~3% fee loss. I know the fee loss is a little bit high... Maybe a Binance Card is a better choice if you could manage the kyc.
